static/js/components/dashboard/CourseDescription_test.js

```diff
@@ -63,6 +227,44 @@ describe('CourseDescription', () => {
     assert.lengthOf(elements.titleLink, 0);
   });
 
+  it('show view course on edX link when non-enroll statues and enrollment_url is set', () => {
```

Comment:
please change to: 'shows an enrollment link for an unenrolled course run with an enrollment url'